api/events/consent.js

```diff
@@ -1,226 +1,230 @@
-      }
-      if (event.payload.consentRecords) {
-        for (const record of event.payload.consentRecords) {
-        // Consent record has multiple organisations
-          for (const org of record.metadata.organisationSecureKeys) {
-            organisations.push(await registry.organizationById(org.legalEntity))
-          }
-          record.organisations = organisations
-        }
-      }
-
-      transactions.push({
-        status: event.name,
-
-        // Only unique organisations
-        organisations: organisations.filter((obj, pos, arr) =>
-          arr.map(mapObj =>
-            mapObj.name
-          ).indexOf(obj.name) === pos
-        )
-      })
-    }
-
-    return transactions
-  } catch (e) {
-    Logger.error(`Error in Nuts node query for consent events: ${e}`)
-  }
-}
-
-async function getReceivedConsents (patient) {
-  try {
-    // Fetch received consents for this patient
-    const consents = await consentStore.consentsFor({
-      subject: patient,
-      actor: config.organisation
-    })
-
-    // Map URNs to sane organisations
-    let organisations = []
-    if (consents.totalResults > 0 && consents.results) { organisations = await urnsToOrgs(consents.results.map(o => o.custodian)) }
-
-    return organisations
-  } catch (e) {
-    Logger.error(`Error in Nuts node query for finding consents: ${e}`)
-  }
-}
-
-async function getGivenConsents (patient) {
-  try {
-    // Fetch confirmed consents
-    const consents = await consentStore.consentsFor({
-      subject: patient,
-      custodian: config.organisation
-    })
-
-    // Fetch consents in transit
-    const pending = await consentInTransit.get({
-      subject: patient.bsn,
-      custodian: config.organisation.agb
-    })
-
-    // Are these consents in transit still pending?
-    const stillPending = []
-    for (const consent of pending) {
-      const currentStatus = await eventStore.getEvent(consent.jobId)
-      if (currentStatus.name !== 'completed') { stillPending.push(consent) }
-    }
-
-    // Map URNs to sane organisations
-    let organisations = []
-    if (consents.totalResults > 0 && consents.results) { organisations = await urnsToOrgs(consents.results.map(o => o.actor)) }
-    if (stillPending.length > 0) {
-      const orgs = await urnsToOrgs(stillPending.map(o => o.actor))
-      organisations = organisations.concat(orgs.map(o => { o.name += ' <em>(pending acceptance)</em>'; return o }))
-    }
-
-    return organisations
-  } catch (e) {
-    Logger.error(`Error in Nuts node query for finding consents: ${e}`)
-  }
-}
-
-async function urnsToOrgs (urns) {
-  const orgs = []
-  for (const urn of urns) {
-    try {
-      orgs.push(await registry.organizationById(urn))
-    } catch (e) {
-      orgs.push({
-        identifier: urn,
-        name: 'Could not find organisation: ' + e
-      })
-    }
-  }
-  return orgs
-}
+// The functionality this file contains is deprecated because it's being reworked in `nuts-node`.
```

Comment:
Then why touch this file at all in this PR? Did it break things?

Reply:
Tested, didn't break anything. Reverted the change.